README.md

```diff
@@ -183,3 +183,39 @@ quarto publish gh-pages --no-browser
 * **Pro-tip** : If you are using VS Code as your code editor, install the [Quarto extension](https://marketplace.visualstudio.com/items?itemName=quarto.quarto) to make editing/previewing the doc site a lot smoother.
 
 
+## ☸️Running in Docker 
+
+We have created a [docker image](https://github.com/butchtm/unicef-ai4d-poverty-mapping/pkgs/container/povmap-jupyter) (`ghcr.io/butchtm/povmap-jupyter`) of the poverty mapping repo for those who want to view the notebooks or rollout the models for new countries and new data (e.g. new nightlights and ookla years)
+
+To run these docker images please copy and the following scripts to run on your linux, mac or windows (wsl) terminals:
+
+* **View Jupyter notebooks (Read-only)** This will run a jupyter notebook environment containing the poverty mapping notebooks at http://localhost:8888/lab/tree/notebooks
+
+```bash
+curl -s https://raw.githubusercontent.com/thinkingmachines/unicef-ai4d-poverty-mapping/main/localscripts/run-povmap-jupyter-notebook.sh > run-povmap-jupyter-notebook.sh && \
+chmod +x run-povmap-jupyter-notebook.sh && \
+./run-povmap-jupyter-notebook.sh
+```
+* **Country-wide rollout** This will run an interactive dialog that will rollout the poverty mapping models for different countries
```

Comment:
To clarify, what does this do? Does it just replicate what we did? I.e. follows the exact rollout configs we had?

Reply:
1) the first script just runs a jupyter notebook environment on docker
2) the second script runs the generate grids and rollout notebooks (provides an interactive dialog prompt to override default parameters - the defaults are the same ones we used to rollout the different countries)
3) the third script is a convenience script to copy the results (both notebooks and data) into a local directory from the docker volume (which persists even after the docker container running the script is gone)